scripts/lib/CIME/SystemTests/pet.py

```diff
@@ -30,6 +30,10 @@ def _case_one_setup(self):
             if self._case.get_value("NTHRDS_%s"%comp) <= 1:
                 self._case.set_value("NTHRDS_%s"%comp, 2)
 
+        # Subtle: machines like titan will not let you overload the number of threads
```

Comment:
@jedwards4b if you double the threads without reducing the tasks, you'd expect to have double the total threads. On a machine that allows you to put 2 threads on a single core, this isn't a problem, but apparently aprun/titan doesn't let you do that. Keep in mind that the num_nodes has already been computed at this point in the case object, so that fact the more threads requires more nodes on titan is lost unless we update by hand like we do here.